lib/active_model/serializer.rb

```diff
@@ -4,15 +4,15 @@
 require 'active_model/serializer/error_serializer'
```

Comment:
you can move this also and rename to `ActiveModelSerializers::Resource` per https://github.com/rails-api/active_model_serializers/pull/1301/files#r42963185 in the RFC in a followup PR maybe?